src/style-sheets/layout/_toolbar.scss

```diff
@@ -1,112 +1,135 @@
-#toolBar {
+.navbar {
+  width: 100%;
+  padding: 0 5%;
   background-color: $white;
-  border-bottom: 2pt solid $olGreen;
-  color: black;
-  height: 50px;
-
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  justify-content: space-between;
 
   position: fixed;
-  top: 0;
-  width: 100%;
   z-index: 1;
+  top: 0;
 
-  & ul.icon-module {
-    width: 30%;
-    // TODO: v-2 display: flex;
-    display: none;
-    flex-direction: row;
+  & ul {
+    margin: 0;
+    padding: 0;
+    display: flex;
     align-items: center;
-    justify-content: flex-end;
-    padding-inline-start: 0px;
-    margin-right: 2%;
-  }
 
-  & li {
-    list-style: none;
-  }
+    & li {
+      margin: 0 2vw;
+      font-size: $olNavLinkSize;
+      text-align: center;
+      font-weight: 200;
+      line-height: 25px;
+
+      list-style-type: none;
 
-  & ul.icon-module li {
-    padding: 0 4px;
+      & a {
+        color: $black;
+        font-family: $olAssistantFont;
+        text-decoration: none;
 
-    & div {
-      padding-top: 3px;
+        &:hover {
+          opacity: 0.7;
+        }
+      }
     }
+  }
 
-    & svg {
-      cursor: pointer;
-      color: $olGreen;
+  & .nav-links {
+    width: 100%;
+    height: 100px;
 
-      &:hover {
-        opacity: 0.7;
+    & .icons {
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+
+      & .logo {
```

Comment:
I think you could shorten this nest to & .logo img {

Reply:
Yeah, I was expecting to have code specific to the `logo` class. It's pretty much unnecessary right now.